### ISSUE TITLE
[Bug]: Unable to install mods

### ISSUE THREAD

**Issue by Icxulub** [TexTools/FFXIV_TexTools_UI#297]
### Expected behavior?

I reinstalled my computer and since and usually before when I downloaded a mod, it was opening a special window where I could modify the details of the mod if it was possible, or just install it,wait a couple seconds,and it says "installatiion successfull" (or something like that) and close the window when I clicked on the final step.

### Actual behavior?

![screen1](https://github.com/TexTools/FFXIV_TexTools_UI/assets/174385307/8087e87a-9ac6-447d-94be-a391cf5a5237)

Every time I open a mod to install it, it just redirect me to the main textool page, as seen in the screenshot above, no matter what I try. When I try to use the repair tool within Textools, it display this : 

![image_2024-07-01_232732946](https://github.com/TexTools/FFXIV_TexTools_UI/assets/174385307/db4daf23-d7d2-47cc-a3bf-5b92e753ef41)

### Steps to reproduce the problem

1.Open a mod .ttmp2 with TexTools
2.TexTools open to its main menu and not the installation menu for the mod
3.
4.
5.
...

### What aspect of FFXIV TexTools were you using, and what Item were you interacting with?

Mod installing

### Has this has been confirmed as an actual bug?

- [X] This has been confirmed to be a legitimate issue and not caused by bad game data.

**Comment by Lunaretic** [comment]
This is not a bug.  See documentation.  Writing to live game files is disabled in safe mode, for (as the name implies) safety reasons.  Use Unsafe mode if you wish to alter your live game files.

**Comment by Icxulub** [comment]
Yes it is what I think I read, and I disabled the safe mod at the beginning and clicked on the option to open it in unsafe mode by default, but I am still confronted by the same problem. I am not very fluent in english, I have a bit of difficulty to find the right documentation, but everything I found by doing some research either here or on discord didn't helped. As I say, I never was confronted by this problem before I reinitialized my computer, so am I a bit confused aswell. Thanks for the response though !
